### PR TITLE
Adjust RBAC API integration test responses

### LINKED ISSUE #9695: Review RBAC agent API integration tests expected response
|Wazuh version|Component|
|---|---|
| 4.2.0-rc13 | API |

After reviewing the API integration tests in the issue https://github.com/wazuh/wazuh/issues/9689, we have seen that there are some RBAC tests that expect certain fields in the response. These tests must only check the status code as they aim to test the RBAC permissions.

For instance, this test:

https://github.com/wazuh/wazuh/blob/39b4ff30a8a6f9b034240798ebbf43f4623ab625/api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml#L238-L266

It should be like this:

```yaml
---
test_name: GET /agents/{agent_id}/stats/{component}

stages:

 - name: Try to get logcollector stats from agent 002 (Allowed)
   request:
     verify: False
     url: "{protocol:s}://{host:s}:{port:d}/agents/002/stats/logcollector"
     method: GET
     headers:
       Authorization: "Bearer {test_login_token}"
   response:
     status_code: 200
```

### PR DESCRIPTION
**Pull request by Kondent**
|Related issue|
|---|
| #9695 |

# Description
This PR closes #9695.
I updated `test_rbac_white_agent_endpoints.tavern.yaml`.
Generic responses were deleted since for testing purposes we only need status codes.

# Integration test:
```
⬦⬦⬦ ~/git/wazuh/api/test/integration ⊶ fix/9695-rbac-integration-response ⨘ pytest -vv test_rbac_white_agent_endpoints.tavern.yaml --disable-warnings
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.5, pytest-5.0.0, py-1.10.0, pluggy-0.13.1 -- /home/kondent/pythonEnv/inttest-env/bin/python3.9
cachedir: .pytest_cache
metadata: {'Python': '3.9.5', 'Platform': 'Linux-5.11.0-25-generic-x86_64-with-glibc2.31', 'Packages': {'pytest': '5.0.0', 'py': '1.10.0', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'metadata': '1.11.0', 'tavern': '1.2.2'}}
rootdir: /home/kondent/git/wazuh/api/test/integration, inifile: pytest.ini
plugins: html-3.1.1, metadata-1.11.0, tavern-1.2.2
collected 41 items                                                                                                                                                                           

test_rbac_white_agent_endpoints.tavern.yaml::GET /agents PASSED                                                                                                                        [  2%]
...
...
...
test_rbac_white_agent_endpoints.tavern.yaml::PUT /agents/node/{node_id}/restart PASSED                                                                                                 [100%]

========================================================================== 41 passed, 1 warnings in 609.43 seconds ===========================================================================
```

### CODE REVIEW

File: api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml [hunk 1]
```diff
@@ -617,16 +581,6 @@ stages:
...
-     json:
-       error: 0
-       data:
-         affected_items:
-           - count: 6
-             os:
-               platform: !anystr
-         failed_items: []
-         total_affected_items: !anyint
-         total_failed_items: 0
```
Comment (vicferpoy) [inline api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml:629]: As you can read in the test name, this test permissions are partially allowed (we are asking for for distinct stats in all agents). This means that the `count` field needs to be `6` exactly, as we have read permissions for 6 agents in this environment.

**Review by vicferpoy — CHANGES_REQUESTED** [review]
Missing changes on RBAC black tests.

**Comment by Kondent** [comment]
# Update
Requested changes were added.

## Int test: test_rbac_black_agent & test_rbac_white_agent
```
↪ ~/git/wazuh/api/test/integration ⊶ fix/9695-rbac-integration-response ⨘ pytest -vv test_rbac_black_agent_endpoints.tavern.yaml --disable-warnings && pytest -vv test_rbac_white_agent_endpoints.tavern.yaml --disable-warnings 
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.5, pytest-5.0.0, py-1.10.0, pluggy-0.13.1 -- /home/kondent/pythonEnv/inttest-env/bin/python3.9
cachedir: .pytest_cache
metadata: {'Python': '3.9.5', 'Platform': 'Linux-5.11.0-27-generic-x86_64-with-glibc2.31', 'Packages': {'pytest': '5.0.0', 'py': '1.10.0', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'metadata': '1.11.0', 'tavern': '1.2.2'}}
rootdir: /home/kondent/git/wazuh/api/test/integration, inifile: pytest.ini
plugins: html-3.1.1, metadata-1.11.0, tavern-1.2.2
collected 41 items                                                                                                                                                                           

test_rbac_black_agent_endpoints.tavern.yaml::GET /agents PASSED                                                                                                                        [  2%]
...
...
test_rbac_black_agent_endpoints.tavern.yaml::PUT /agents/node/{node_id}/restart PASSED                                                                                                 [100%]

========================================================================== 41 passed, 1 warnings in 632.29 seconds ===========================================================================
==================================================================================== test session starts =====================================================================================
platform linux -- Python 3.9.5, pytest-5.0.0, py-1.10.0, pluggy-0.13.1 -- /home/kondent/pythonEnv/inttest-env/bin/python3.9
cachedir: .pytest_cache
metadata: {'Python': '3.9.5', 'Platform': 'Linux-5.11.0-27-generic-x86_64-with-glibc2.31', 'Packages': {'pytest': '5.0.0', 'py': '1.10.0', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'metadata': '1.11.0', 'tavern': '1.2.2'}}
rootdir: /home/kondent/git/wazuh/api/test/integration, inifile: pytest.ini
plugins: html-3.1.1, metadata-1.11.0, tavern-1.2.2
collected 41 items                                                                                                                                                                           

test_rbac_white_agent_endpoints.tavern.yaml::GET /agents PASSED                                                                                                                        [  2%]
...
...
test_rbac_white_agent_endpoints.tavern.yaml::PUT /agents/node/{node_id}/restart PASSED                                                                                                 [100%]

========================================================================== 41 passed, 1 warnings in 607.30 seconds ===========================================================================
```

**Comment by Kondent** [comment]
# Update
Im closing this PR since I had some problems trying to change root branch.
New PR: `9904`

**Pull request closed without merging**